### DISCUSSION TITLE
Uploading Non Images

### DISCUSSION THREAD

**Discussion by trantechsolutions** [POPWorldMedia/POPForums#330] [Q&A] [answered]
Is there a functionality or a roadmap to allow for uploading of any microsoft extensions (.docx, .xlsx, etc) or pdfs?

**Comment by jeffputz** [comment]
Nothing on the roadmap, but that would be a fairly straightforward thing to do now that the image stuff is in place. Do you imagine that the files are associated with posts or topics? Or both?

**Comment by trantechsolutions** [comment]
Associated to post. Also are subcategories a thing?We are migrating off NodeBB, so we are trying to simulate a lot of the functionality.On Apr 12, 2023 11:04 AM, Jeff Putz ***@***.***> wrote:
Nothing on the roadmap, but that would be a fairly straightforward thing to do now that the image stuff is in place. Do you imagine that the files are associated with posts or topics? Or both?

**Reply by jeffputz** [reply, marked as answer]
No, no subcategories. Why don't you create an issue titled something like, "Add file attachment capability to posts."
